src/components/Menu.js

```diff
@@ -9,18 +19,33 @@ import { NAVIGATION } from '../lib/constants';
 import LangSelect from './LangSelect';
 
 function Menu({ locale }) {
+  let location = useLocation();
+  let history = useHistory()
+
   const [expanded, setExpanded] = useState(false);
   const { t } = useTranslation();
 
   const onClick = () => {
     setExpanded(prvExpanded => !prvExpanded);
   }
 
+  const getNewUrl = (page) => {
+    const curPath = location.pathname;
+    const lng = curPath.split('/')[1];
+    console.log(`/${lng}/${page}`);
+    return `/${lng}/${page}`;
+  }
+
   const menuItems = NAVIGATION.map(({name, route}) => {
+    const handleClick = event => {
+      history.push(getNewUrl(name));
+      event.preventDefault();
+    }
+
     return (
-      <Link href={route} key={t(name)} className="usa-nav__link">
+      <Button onClick={handleClick} key={t(name)} className="usa-nav__link">
         <span>{t(name)}</span>
-      </Link>
+      </Button>
```

Comment:
Not sure what this is for, but re-evaluate to see if it should be a Button or Link.